content/pages/home.md

```diff
@@ -5,39 +5,42 @@ missionStatement: |-
-    and work with the community of external organizations supporting the Operation
-    _Allies Welcome_ mission to help facilitate aid deliveries. Once aid arrives on
-    base we work with the US Marines to distribute items to guests, and supply materials
-    for NGOs running education & activity programs to use in their classes. In November
-    & December, our efforts resulted in more than 750,000 items being distributed
-    to or used by 7,500 guests at Fort Pickett. You can support our work by [donating
-    here](https://opencollective.com/fort-pickett)!
-
+  - title: Refugee Aid
+    location: Europe
+    description:
+      'Distribute Aid helps European grassroots refugee aid groups support
+      over 100,000 asylum seekers. We assist with every point in the supply chain: assessing
+      needs, gathering supplies at home or from in-kind donors, coordinating shipments,
+      and advising on distributions. We currently send several regular shipments each
+      month to multiple destinations and handle specialty/emergency response shipments
+      as well.'
+  - title: COVID-19 Response
+    location: Europe & USA
+    description:
+      Distribute Aid is actively working to prevent the spread of COVID-19
+      globally. We regularly send hygiene and PPE shipments to our European partners
+      who continue to work to protect people in crowded refugee camps. Because COVID-19
+      affects everybody, our response grew as the virus spread. We now also collaborate
+      with multiple grassroots networks in the US to support their response.
+  - title: In-kind Donations
+    location: Global
+    description:
+      Distribute Aid runs a successful in-kind donation programme where we
+      connect companies with aid organisations. Through our network of hundreds of charities,
+      we can handle coordinating large volumes of donations. To date, we have secured
+      over 5 million items through our in-kind donor programme. Email [donate-aid@distributeaid.org](mailto:donate-aid@distributeaid.org
+      "Contact Distribute Aid") if you can donate in-kind.
+  - title: Fort Pickett
+    location: USA
+    description:
+      Distribute Aid was officially invited to join Operation _Allies Welcome_
+      at Fort Pickett in November. Our role on base is to manage all the donated aid.
+      We work with NGOs and departments across the base to assess the needs of our guests,
+      and work with the community of external organizations supporting the Operation
+      _Allies Welcome_ mission to help facilitate aid deliveries. Once aid arrives on
+      base we work with the US Marines to distribute items to guests, and supply materials
+      for NGOs running education & activity programs to use in their classes. In November
+      & December, our efforts resulted in more than 750,000 items being distributed
+      to or used by 7,500 guests at Fort Pickett. You can support our work by [donating to us through our
```

Comment:
The only change in this file is the anchor text for this link here... the rest is probably prettier doing some prettifying... 🤨